packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -293,15 +312,23 @@ describe('TokenDetectionController', () => {
       const mockGetBalancesInSingleCall = jest.fn().mockResolvedValue({
         [sampleTokenA.address]: new BN(1),
       });
-      const selectedAddress = '0x0000000000000000000000000000000000000001';
+      const selectedAccount = createMockInternalAccount({
+        address: '0x0000000000000000000000000000000000000001',
+      });
```

Comment:
A lot of repeated code that could be done during the setup of the tests, do you think it can be simplified?

Reply:
The selected address was converted to an internal account here. I could have created a default mock account and used it for all the tests but i wanted to preserve the address that was previously used for the test.